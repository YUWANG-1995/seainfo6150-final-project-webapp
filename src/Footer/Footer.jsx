import React from "react";
import styles from "./Footer.module.css";

const Footer = () => {
  return (
    <footer className={styles.footerSytle}>
      <p>Author: Yu Wang <br/>
      @All the code written by Yu Wang<br/>
      @All images and content are just for Studying without commerical use</p>
    </footer>
  )
}

export default Footer;
