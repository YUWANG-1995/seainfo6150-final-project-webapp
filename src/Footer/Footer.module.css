.footerSytle{
  text-align: center;
  font-family: 'Open Sans', sans-serif;
  font-size:10px;
  font-weight: lighter;

  position:absolute;
  left:0;
  bottom:0;
  right:0;
}
