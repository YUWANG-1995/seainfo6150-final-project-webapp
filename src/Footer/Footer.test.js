import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer.jsx";

it("renders the Footer Component Correctly", () => {

  const { container } = render(
      <Footer />
  );
  expect(container).toMatchSnapshot();
});
