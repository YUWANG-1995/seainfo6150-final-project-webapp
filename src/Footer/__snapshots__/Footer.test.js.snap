// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Footer Component Correctly 1`] = `
<div>
  <footer
    class="footerSytle"
  >
    <p>
      Author: Yu Wang 
      <br />
      @All the code written by Yu Wang
      <br />
      @All images and content are just for Studying without commerical use
    </p>
  </footer>
</div>
`;

exports[`renders the Header Component Correctly 1`] = `
<div>
  <footer
    class="footerSytle"
  >
    <p>
      Author: Yu Wang 
      <br />
      @All the code written by Yu Wang
      <br />
      @All images and content are just for Studying without commerical use
    </p>
  </footer>
</div>
`;
